### DISCUSSION TITLE
How do I change my payment method

### DISCUSSION THREAD

**Discussion by mcosciel36** [community/community#42868] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I can't find where I can update my credit card method.

**Comment by mcosciel36** [comment]
This link suggests I should see a sidebar.   I do not see a sidebar. 
https://docs.github.com/en/billing/managing-your-github-billing-settings/adding-or-editing-a-payment-method#updating-your-personal-accounts-payment-method

If I click on one of my projects and then "settings" I see an "access" section in the sidebar but the only thing underneath it is "collaborators".   There's nothing mentioning billing.

**Comment by mcosciel36** [comment, marked as answer]
I found the sidebar, "Access" and "Billing and Plans" by clicking on my icon at the top right hand corner and clicking on "settings".

The document above doesn't mention how to get to the sidebar.

**Reply by mrityunjaySequelstring** [reply]
Hey,

I mistakenly paid the amount to **Laya.Yesodharan@sequelstring.com**, but it needs to be updated to **sequelstring-com**.

Please assist with this.
